### ISSUE TITLE
v3.8.0 Error: Failed to launch the browser process

### ISSUE THREAD

**Issue by giuliomagnifico** [grafana/grafana-image-renderer#450]
**What happened**:

If I upgrade my linux machine to the latest release I can't use grafana-image-render because I got this error

```
Aug 26 09:27:50 Grafana node[1773]: {"level":"error","message":"Request failed","stack":"Error: Failed to launch the browser process!\n/root/.cache/puppeteer/chrome/l
inux-116.0.5845.96/chrome-linux64/chrome: 3: Syntax error: \")\" unexpected\n\n\nTROUBLESHOOTING: https://pptr.dev/troubleshooting\n\n    at Interface.onClose (/root/
grafana-image-renderer/node_modules/puppeteer-core/node_modules/@puppeteer/browsers/lib/cjs/launch.js:277:24)\n    at Interface.emit (node:events:526:35)\n    at Inte
rface.close (node:internal/readline/interface:533:10)\n    at Socket.onend (node:internal/readline/interface:259:10)\n    at Socket.emit (node:events:526:35)\n    at 
endReadableNT (node:internal/streams/readable:1359:12)\n    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)","url":"/render?deviceScale
Factor=2.000000&domain=192.168.1.6&encoding=&height=1510&renderKey=sONjIlKMgdgNRMmoTutLmo68fKMCkMMv&timeout=60&timezone=&url=http%3A%2F%2F192.168.1.6%3A3001%2Fd%2FdH3
ws5OVk%2Fcasa%3Fwidth%3D1200%26height%3D1510%26kiosk%3Dtv%26scale%3D2%26from%3Dnow-24h%26to%3Dnow%26theme%3Ddark%26from%3Dnow-30m%26to%3Dnow%26render%3D1&width=1200"}
Aug 26 09:27:50 Grafana node[1773]: {"level":"error","message":"::ffff:192.168.1.6 - - [26/Aug/2023:07:27:50 +0000] \"GET /render?deviceScaleFactor=2.000000&domain=19
2.168.1.6&encoding=&height=1510&renderKey=sONjIlKMgdgNRMmoTutLmo68fKMCkMMv&timeout=60&timezone=&url=http%3A%2F%2F192.168.1.6%3A3001%2Fd%2FdH3ws5OVk%2Fcasa%3Fwidth%3D1
200%26height%3D1510%26kiosk%3Dtv%26scale%3D2%26from%3Dnow-24h%26to%3Dnow%26theme%3Ddark%26from%3Dnow-30m%26to%3Dnow%26render%3D1&width=1200 HTTP/1.1\" 500 96 \"-\" \"
Grafana/10.1.0\"\n"}
```

I understood that is something related to the new *puppeteer* version, but I'm not understing how to fix it.

thanks for the help

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Grafana Image Renderer version: 3.8.0
- Grafana version: 10
- Installed plugin or remote renderer service:
- OS Grafana Image Renderer is installed on: Raspbian/DietPI on Raspberry PI 
- User OS & Browser:
- Others:

**Comment by jlcrow** [comment]
Same problem in Kubernetes 1.26 GKE cluster, woke up to a crashlooping pod this morning - thanks to the Latest tag for the image renderer in the grafana helm chart.

{"level":"info","maxConcurrency":10,"message":"using clustered browser","mode":"context","timeout":30}
{"level":"info","message":"HTTP Server started, listening at http://0.0.0.0:8081"}
Error: Unable to launch browser, error message: Failed to launch the browser process!
chrome_crashpad_handler: --database is required
Try 'chrome_crashpad_handler --help' for more information.
[18:18:0826/105029.115277:ERROR:socket.cc(120)] recvmsg: Connection reset by peer (104)


TROUBLESHOOTING: https://pptr.dev/troubleshooting

    at Cluster.<anonymous> (/usr/src/app/node_modules/puppeteer-cluster/dist/Cluster.js:119:23)
    at Generator.throw (<anonymous>)
    at rejected (/usr/src/app/node_modules/puppeteer-cluster/dist/Cluster.js:6:65)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)

**Comment by Shelestov7** [comment]
I have the same issue

**Comment by lg-d** [comment]
+1, running Kubernetes 1.24 on EKS, tagged image to 3.7.2 which has helped

**Comment by booleanbetrayal** [comment]
Seeing this as well on 3.8.0

**Comment by AgnesToulet** [comment]
Hello! The original issue is a duplicate of https://github.com/grafana/grafana-image-renderer/issues/449 and has a fix in progress: https://github.com/grafana/grafana-image-renderer/pull/451

The ones on Kubernetes are different though. Could you open a new issue and share your configuration? Also, could you set your log level to "debug" (using the `LOG_LEVEL` env variable) to see if you get more logs? Thanks!

**Comment by nickjmv** [comment]
Same issue here on AWS EKS v1.27. Pinning the image tag to 3.7.1 for the image renderer is a fix!

**Comment by beatkind** [comment]
Same issue here

**Comment by alirezamansoori** [comment]
Will v3.8.1 be released with the fix from #449 ?

**Comment by giuliomagnifico** [comment]
Just update to 3.8.1 but I got the same error:

```
{"level":"error","message":"Request failed","stack":"Error: Failed to launch the browser process!
\n/root/.cache/puppeteer/chrome/linux-116.0.5845.96/chrome-linux64/chrome: 3: Syntax error: \")\" unexpected\n\n\nTROUBLESHOOTING: http
s://pptr.dev/troubleshooting\n\n    at Interface.onClose (/root/grafana-image-renderer/node_modules/puppeteer-core/node_modules/@puppet
eer/browsers/lib/cjs/launch.js:277:24)\n    at Interface.emit (node:events:526:35)\n    at Interface.close (node:internal/readline/inte
rface:533:10)\n    at Socket.onend (node:internal/readline/interface:259:10)\n    at Socket.emit (node:events:526:35)\n    at endReadab
leNT
```

**Comment by AgnesToulet** [comment]
Fixed in https://github.com/grafana/grafana-image-renderer/pull/459, the fix was released in 3.8.2.
